backend/.htaccess

```diff
@@ -0,0 +1,11 @@
+<IfModule mod_rewrite.c>
+
+RewriteEngine on
+
+
+RewriteBase /~tbrooks/ACM-Manager/backend
+RewriteCond %{REQUEST_FILENAME} !-f
+RewriteCond %{REQUEST_FILENAME} !-d
+RewriteRule ^(.*) /~tbrooks/ACM-Manager/backend/server.php [L]
```

Comment:
Should be acmuser. Must not have synced back down since it is a hidden file. This has been fixed.